### PR TITLE
javadoc for signURL for blob with generation

### PR DESCRIPTION
**Pull request by dmitry-fa**

### CODE REVIEW

File: google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java [hunk 1]
```diff
@@ -2498,6 +2498,19 @@ Blob create(
...
+   *
+   * <pre>{@code
+   * String bucketName = "my-unique-bucket";
+   * String blobName = "my-blob-name";
+   * long generation = 1576656755290328L;
+   *
+   * URL signedUrl = storage.signUrl(
+   *     BlobInfo.newBuilder(bucketName, blobName, generation).build(),
+   *     7, TimeUnit.DAYS,
+   *     SignUrlOption.withQueryParams(ImmutableMap.of("generation", "" + generation)));
```
Comment (rahulKQL) [inline google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java]: `SignUrlOption.withQueryParams(ImmutableMap.of("generation", "" + generation)));` => ` SignUrlOption.withQueryParams(ImmutableMap.of("generation",  generation)));`

**Review by rahulKQL — CHANGES_REQUESTED** [review]

File: google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java (hunk 1 above)
Comment (dmitry-fa) [reply google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java]: it won't compile, the type of 'generation' is long it should be coverted to String

Comment (rahulKQL) [reply google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java]: Ahh, My bad I assume it to be string without checking.

Still, I think `SignUrlOption.withQueryParams(ImmutableMap.of("generation", String.valueOf(generation))));` is more readable

Comment (dmitry-fa) [reply google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java]: okay, fixed

**Review by rahulKQL — APPROVED** [review]

**Pull request closed without merging**